.changelog/unreleased/breaking-changes/2890-proto-privval-get-pubkey-response-public-key.md

```diff
@@ -0,0 +1,2 @@
+- [`proto/privval`]  Replace `pub_key` with `pub_key_type` and `pub_key_bytes` in
+  `PubKeyResponse` ([\#2890](https://github.com/cometbft/cometbft/pull/2890))
```

Comment:
Since we have an issue (#2878), I think it's better to reference the issue here

Reply:
I thought the convention was to link the PR, not the issue?

Reply:
We try to link the issue, if there's one.